scripts/firedrake-zenodo

```diff
@@ -97,6 +93,13 @@ else:
                         format='%(message)s')
 log = logging.getLogger()
 
+cwd = os.getcwd()
+try:
+    src = os.environ["VIRTUAL_ENV"] + "/src"
```

Comment:
I think that the install procedure would have to encode where it was installed *from*.